### DISCUSSION TITLE
模板中图片添加引用导致顺序错误

### DISCUSSION THREAD

**Discussion by CSberlin** [stone-zeng/fduthesis#310] [Q&A] [answered]
在第三章图片的\caption{}标签内使用\cite{参考文献}, 导致该参考文献到了引用的第一个位置(顺序被调前), 请问如何解决? 或者说应该在图片的哪个标签内加入引用合适?

**Comment by stone-zeng** [comment, marked as answer]
用 `\caption[短标题]{长标题}` 这种写法，`\cite` 只放在 `长标题` 里面就行了。
